pages/_app.tsx

```diff
@@ -51,6 +51,17 @@ function MyApp<MyAppProps>({ Component, pageProps }) {
   const formMethods = useForm<FormInputData>({ mode: "onBlur" });
   const { favIconPath, appTitle } = appConfig;
 
+  // Setting the "lang" and the "dir" attribute
+  const { i18n } = useTranslation("common");
+  React.useEffect(() => {
+    let lang = query.lang !== undefined ? query.lang : "en";
+    if (lang === "zhcn") {
+      lang = "zh-cn";
+    }
+    document.getElementById("__next").dir = `${i18n.dir()}`;
```

Comment:
This is fine but the footer is in RTL. It's actually not a big deal so we can make that a new ticket for part of a refactor.